sympy/physics/mechanics/joint.py

```diff
@@ -1399,9 +1399,26 @@ class PlanarJoint(Joint):
     ($\\vec{v}_1$, $\\vec{v}_2$) and generalized coordinates ($q_1$, $q_2$),
     i.e. $\\vec{r} = q_1 \\hat{v}_1 + q_2 \\hat{v}_2$. The direction cosine
     matrix between the ``child_interframe`` and ``parent_interframe`` is formed
-    using a simple rotation ($q_0$) about the rotation axis. The page on the
-    joints framework provides a more detailed explanation of the intermediate
-    frames.
+    using a simple rotation ($q_0$) about the rotation axis.
+    In order to simplify the definition of the ``PlanarJoint`` the
```

Comment:
Should there be a blank line between these two paragraphs?

Reply:
Yeah, wasn't sure thought it may improve readability (but like this it will just be one big paragraph. Will add an extra blank line.